compiler/rustc_codegen_ssa/src/mir/block.rs

```diff
@@ -641,67 +641,73 @@ impl<'a, 'tcx, Bx: BuilderMethods<'a, 'tcx>> FunctionCx<'a, 'tcx, Bx> {
             return;
         }
 
-        if intrinsic.is_some() && intrinsic != Some(sym::drop_in_place) {
-            let intrinsic = intrinsic.unwrap();
-            let dest = match ret_dest {
-                _ if fn_abi.ret.is_indirect() => llargs[0],
-                ReturnDest::Nothing => {
-                    bx.const_undef(bx.type_ptr_to(bx.arg_memory_ty(&fn_abi.ret)))
-                }
-                ReturnDest::IndirectOperand(dst, _) | ReturnDest::Store(dst) => dst.llval,
-                ReturnDest::DirectOperand(_) => {
-                    bug!("Cannot use direct operand with an intrinsic call")
-                }
-            };
+        match intrinsic {
+            None | Some(sym::drop_in_place) => {}
```

Comment:
Doesn't `drop_in_place` no longer get codegened now? The `Self::codegen_intrinsic_call` call is inside the `Some(intrinsic)` match arm.

Reply:
I don't think it was codegened previously, there was an explicit check for `intrinsic != Some(sym::drop_in_place)`, and it's not referenced inside of `fn codegen_intrinsic_call(..)`.